lib/index.ts

```diff
@@ -42,3 +42,24 @@ Cypress.Commands.add('percySnapshot', (name: string, options: any = {}) => {
     })
   })
 })
+
+
+// An attempt to resiliently find the path to the 'percy-healthcheck' script, and to do so
+// in a cross-platform manner.
+function _healthcheckPath() {
+  try {
+    // Try to resolve with respect to the install local module.
+    return require.resolve('@percy/cypress/dist/percy-healthcheck')
```

Comment:
I don't think this path will be correct. When we `bin` something in the `package.json` it's put in the `./node_modules/.bin/` directory:
![image](https://user-images.githubusercontent.com/2072894/55113466-5230b980-50ad-11e9-9767-58751cc9ec9d.png)


I don't see it in the `dist` dir: 
![image](https://user-images.githubusercontent.com/2072894/55113426-37f6db80-50ad-11e9-906f-f1667f7c5227.png)

But it is in the project root: 
![image](https://user-images.githubusercontent.com/2072894/55113456-4644f780-50ad-11e9-8075-7318a3b525b5.png)

Reply:
I changed the location of the file, and it is no longer in the `bin` part of the `package.json`, so I think this is correct.

My original reasoning for putting it in `bin` was that it would let us rely on node finding the correct path to it, but that doesn't seem to work well, so I'm reverting to a more direct approach.

Reply:
Ahhhhhhhhhhhhhh, you're absolutely right. Never mind!